.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Versions
+<!--
+In general, please make sure you are using the latest versions of software when reporting an issue. If you are not using the latest JupyterLab, please be sure to install the right labextension: https://github.com/matplotlib/ipympl#install-an-old-jupyterlab-extension
+
+Please include the result of the following command, which will print out the version of python, ipympl and the Jupyter ecosystem.
+
+```bash
+python -c "import sys; print(sys.version); import ipympl; print('ipympl version:', ipympl.__version__)" && jupyter --version && jupyter nbextension list && jupyter labextension list
```

Comment:
```
python -c "import sys; print('\n',sys.version); import ipympl; print('ipympl version:', ipympl.__version__)" && jupyter --version && jupyter nbextension list && jupyter labextension list
```
I added a newline at the beginning. Makes it a little easier to copy for people who are new to command line.